### DISCUSSION TITLE
Songs not going order of the beatmap collection

### DISCUSSION THREAD

**Discussion by kujubuo** [ppy/osu#14944] [Q&A] [answered]
I opened osu lazer so i can listen something and i put my song collection there and it wasnt going the order of the collection and it was making me listen other stuff that isnt on my collection. I am not sure if this is a bug or not

**Comment by bdach** [comment]
please upload a video of this.

**Comment by kujubuo** [comment, marked as answer]
https://user-images.githubusercontent.com/91714989/135765516-62419a7c-f798-4163-9450-0567d89124f6.mp4

**Reply by bdach** [reply]
https://github.com/ppy/osu/issues/10507#issuecomment-708893553
